### ISSUE TITLE
Screen Readers Cannot Announce Error Messages for Empty Required Fields

### ISSUE THREAD

**Issue by helenliao1** [FreedomScientific/VFO-standards-support#713]
### Summary

> When there is an error in empty required fields, screen reader is not announcing the error message.

### Expected result

> Screen reader should announce the error message for empty fields

### Actual result

> Screen reader is skipping the error message

**Comment by JAWS-test** [comment]
When reading with the virtual cursors (arrow key navigation), the error message is output correctly.

It is not output for TAB navigation. This is not an error of JAWS, but of the source code. In order to output the message, the field and message must be linked, e.g. by aria-errormessage (https://w3c.github.io/aria/#aria-errormessage)
